docs/developer-guide/mapstore-migration-guide.md

```diff
@@ -22,7 +22,75 @@ This is a list of things to check if you want to update from a previous version
 
 ## Migration from 2022.01.00 to 2022.02.00
 
+### Support for OpenID
```

Comment:
why not putting this in the section starting from latest minor?

Migration from 2022.01.02 to 2022.02.00

Reply:
I reused the section that was already present. Going to fix it